src/sentry/static/sentry/app/views/organizationIntegrations/index.jsx

```diff
@@ -1,75 +1,141 @@
-import {Box} from 'grid-emotion';
+import {Box, Flex} from 'grid-emotion';
+import {keyBy} from 'lodash';
 import PropTypes from 'prop-types';
 import React from 'react';
 import styled from 'react-emotion';
 
+import {
+  Panel,
+  PanelBody,
+  PanelHeader,
+  PanelItem,
+  PanelItemGroup,
+} from 'app/components/panels';
+import {
+  addErrorMessage,
+  addLoadingMessage,
+  addSuccessMessage,
+} from 'app/actionCreators/indicator';
+import {openIntegrationDetails} from 'app/actionCreators/modal';
+import {sortArray} from 'app/utils';
 import {t} from 'app/locale';
-import {Panel, PanelBody, PanelHeader, PanelItem} from 'app/components/panels';
 import AsyncComponent from 'app/components/asyncComponent';
 import Button from 'app/components/buttons/button';
+import InstalledIntegration from 'app/views/organizationIntegrations/installedIntegration';
 import Link from 'app/components/link';
+import LoadingIndicator from 'app/components/loadingIndicator';
 import PluginIcon from 'app/plugins/components/pluginIcon';
 import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
-import theme from 'app/utils/theme';
-
-const ProviderName = styled.div`
-  font-size: 16px;
-  font-weight: bold;
-  margin-bottom: 3px;
-`;
-
-const AuthorName = styled.div`
-  color: ${p => p.theme.gray2};
-  font-size: 14px;
-`;
 
 export default class OrganizationIntegrations extends AsyncComponent {
-  static propTypes = {
-    linkPrefix: PropTypes.string,
-    hideHeader: PropTypes.bool,
+  static contextTypes = {
+    organization: PropTypes.object.isRequired,
   };
 
+  // Some integrations require visiting a different website to add them. When
+  // we come back to the tab we want to show our integrations as soon as we can.
+  reloadOnVisible = true;
```

Comment:
We should figure out why acceptance tests were failing with this

Reply:
Ah yeah, looping back on this now